src/invest_reports/sdr_ndr_utils.py

```diff
@@ -1,13 +1,21 @@
 # Utils shared by SDR and NDR
 # (to be extended to support other similar models, and renamed as appropriate)
 
+from collections import namedtuple
+import os
+
 import geopandas
 import pandas
 
+from natcap.invest.spec import ModelSpec
+
 from invest_reports.utils import RasterPlotConfig
 
 TABLE_PAGINATION_THRESHOLD = 10
 
+RasterPlotCaptionGroup = namedtuple(
+    'RasterPlotCaptionGroup', ['inputs', 'outputs', 'intermediates'])
```

Comment:
I wonder if we will ever want/need any other sections besides these three. It could be nice for a report author to decide what the sections are for their model. But I think since this meets the current need so let's go with it.